src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipeEventDispatcher.cs

```diff
@@ -110,7 +110,7 @@ private void CommitDispatchConfiguration()
                 new EventPipeProviderConfiguration(NativeRuntimeEventSource.EventSourceName, (ulong) aggregatedKeywords, (uint) highestLevel, null)
             };
 
-            m_sessionID = EventPipeInternal.Enable(null, 1024, providerConfiguration, 1);
+            m_sessionID = EventPipeInternal.Enable(null, EventPipeSerializationFormat.NetTrace, 1024, providerConfiguration, 1);
```

Comment:
Should this be conditional on Config_NetTraceFormat?

Reply:
In this case its actually arbitrary because this session only feeds the EventListener. It will never be serialized to disk but I didn't want to add the extra code to create another overload of this method that doesn't take the param.